platform/src/components/aws/remix.ts

```diff
@@ -435,7 +435,7 @@ export class Remix extends Component implements Link.Linkable {
           const assetsPath = isUsingVite
             ? path.join("build", "client")
```

Comment:
I think this should actually use the [buildDirectory](https://remix.run/docs/en/main/file-conventions/vite-config#builddirectory) property 🤔

Reply:
This is handled in #736